edx_proctoring/templates/practice_exam/error.html

```diff
@@ -1,28 +1,51 @@
 {% load i18n %}
 <div class="failure sequence proctored-exam" data-exam-id="{{exam_id}}">
   <h3>
-    {% blocktrans %}
-      There was a problem with your practice proctoring session
-    {% endblocktrans %}
+    {% if provider_uses_onboarding %}
```

Comment:
I don't think I saw these changes in the copy document. I know it's just switching "practice" for "onboarding", but it might be worth putting it in there for the sake of completeness.

Reply:
Added [here (link for internal edX folks only, apologies folks)](https://docs.google.com/a/edx.org/document/d/1xOB4v_YsGpcE9qL7sBi95OIhu5vTR0qhTYlh4_bCyO8/edit?disco=AAAACgydMvQ)